internal/build/phases.go

```diff
@@ -38,7 +30,7 @@ func (l *Lifecycle) Create(
 	phaseFactory PhaseFactory,
 ) error {
 	flags := []string{
-		"-cache-dir", cacheDir,
+		"-cache-dir", l.mountPaths.cacheDir(),
```

Comment:
I'm surprised this didn't prompt a bunch of changes in phases_test.go - did I miss something?

Reply:
Hmm... I'll have to look into it. I'm unsure off the top of my head why `phases_test.go` isn't failing for a Windows dockerhost. Will investigate.

Reply:
OK, looked into this. Those tests don't actually start containers, so they pass for any type of dockerhost. There aren't any window-specific changes that need to be tested in there, as it's mostly checking that the proper `mountPaths` functions were called. The differences in `mountPaths` behavior for each dockerhost OS is covered by the acceptance tests.

You could perhaps make an argument for unit testing `mountPaths` in a whitebox test, but it wouldn't increase coverage per se.